scripts/release-stats.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-RELEASE_CHECK=$(node ./scripts/check-is-release.js 2> /dev/null || :)
-if [[ $RELEASE_CHECK == "new-release" || $RELEASE_CHECK == v* ]];
+if [[ $(node ./scripts/check-is-release.js 2> /dev/null || :) = v* ]];
```

Comment:
Can't remember Bash. Should this not be double-equal like before?
```suggestion
if [[ $(node ./scripts/check-is-release.js 2> /dev/null || :) == v* ]];
```

Reply:
Maybe don't inline it and just delete the first branch like you did in other places.

Reply:
Both `=` and `==` works, was clean revert of the change:

https://github.com/vercel/next.js/pull/79611/files#diff-5033399b99c651e66c839af98886bbc869fffa85f5bab79b07d3b0aa440b6d72L3